### DISCUSSION TITLE
АЧЁТАКМОЖНОБЫЛО

### DISCUSSION THREAD

**Discussion by Yebock** [ilyhalight/voice-over-translation#565] [Общее]
То есть всё то время, когда сайт не даёт перевести видео, если нажать по нему правой кнопкой и выбрать пункт "открыть в другой вкладке" его можно было перевести? И это все знали? 
Моего сайта нету в списке и видимо, он использует свой плеер, возможно стоит вписать куданитьтам эту инфу я хз ; /

**Comment by itshardtopickalogin** [comment]
Насколько понимаю, возможность перевода по прямой ссылке появилась относительно недавно. К тому же получить прямую ссылку на видео далеко не всегда просто: иногда сам плеер не дает ее получить (вместо контекстного меню браузера выдает свое, где такой команды может не быть), а если видео потоковое, на уровне ссылки это уже не .mp4, а .m3u8

P.S. Также по опыту даже при наличии прямой ссылки перевод иногда выдает ошибку (как бывает и в ВК, так что это, наверное, скорее уже на стороне Яндекса)
